### PR TITLE
Updated implementation of notification editing (with few fixes)

### PR DESCRIPTION
**Pull request by PubliusGlycia**
[fix] Fixed bug where too long sentences overflow modal window
[fix] Fixed button position
[add] Added notification editing option in modal window

### CODE REVIEW

File: app/javascript/packs/Notification/notification.jsx [hunk 1]
```diff
@@ -91,28 +113,48 @@ export default class Notification extends React.Component {
...
-                                <Col md={1}>{this.props.importance}</Col>
+                                <Col md={6} style={{overflow: "hidden"}}>
+                                <Handler maxLength='30' edit={edit} value={this.props.title} 
+                                    onChange={e =>{
+                                      this.props.setTitle(e.target.value)                                     
+                                    }}>
+                                </Handler></Col>
+                                <Col md={1}>
+                                <div>Pilność</div>
+                                    <Handler edit={edit} value={this.props.importance}
```
Comment (Guciio) [inline app/javascript/packs/Notification/notification.jsx]: Dodać funkcjonalnosc klik =>important noClick =>trival

File: app/javascript/packs/Notification/notification.jsx [hunk 2]
```diff
@@ -155,4 +208,22 @@ export default class Notification extends React.Component {
             </>
         )
     }
-}
\ No newline at end of file
+}
+
+class Handler extends React.Component {
```
Comment (Guciio) [inline app/javascript/packs/Notification/notification.jsx]: Wywalic do folderu np. edit components czy cos żeby nie byly w tej samej klasie

File: app/javascript/packs/Notification/notifications_list.jsx [hunk 3]
```diff
@@ -70,19 +70,33 @@ export default class NotificationList extends React.Component {
         this.fetchPostEvents();
     }
 
+    updateElement = (defect, key, value) => {
```
Comment (Guciio) [inline app/javascript/packs/Notification/notifications_list.jsx]: Nie ma update do supply

File: app/controllers/post_events_controller.rb [hunk 4]
```diff
@@ -56,6 +54,12 @@ def destroy
     end
   end
 
+  # UPDATE
+  def update
+    post_event = PostEvent.find(params[:id])
+    post_event.update_attributes(post_event_params)
```
Comment (hirokiraj) [inline app/controllers/post_events_controller.rb]: we can use just `update` here

Comment (hirokiraj) [reply app/controllers/post_events_controller.rb]: no errors handling?

Comment (hirokiraj) [reply app/controllers/post_events_controller.rb]: does it render anything?

File: app/javascript/packs/Notification/notification.jsx (hunk 1 above)
Comment (PubliusGlycia) [reply app/javascript/packs/Notification/notification.jsx]: functionality added

**Comment by PubliusGlycia** [comment]
Changes applied in following PR: https://github.com/PubliusGlycia/BinarApplication/pull/27
All non-handled fixes will be added with general code refactoring

**Pull request closed without merging**